tests/test_github_pagination.py

```diff
@@ -21,15 +21,38 @@ def get_properties(self, original: bool = True):
         return return_value
 
     def test_run(self):
-        # page size for "pull_requests"
+        
+        streams_to_test = self.expected_streams()
+
+        # Pagination is not supported for "team_memberships" by Github API.
+        # Skipping "teams" stream as it's RECORD count is <= 30.
```

Comment:
Can't we create more than 30 teams?

Reply:
We can't add more teams as it is handled at the organization level.